resource_customizations/jetstream.nats.io/Account/health.lua

```diff
@@ -0,0 +1,34 @@
+hs = {}
+local healthy_cons = { Created=true, Updated=true, Noop=true, SkipCreate=true, SkipUpdate=true }
+if obj.status ~= nil then
+    if obj.status.conditions ~= nil then
+    for i, condition in ipairs(obj.status.conditions) do
+        if condition.status == "False" then
+        hs.status = "Degraded"
```

Comment:
Can you move this to the right, please? And similar to the lines below. Or, let me know if Github UI is being weird on my side.